components/match2/commons/starcraft_starcraft2/match_summary_starcraft.lua

```diff
@@ -62,31 +64,59 @@ function StarcraftMatchSummary.MatchSummary(props)
 		:addClass('brkts-popup')
 		:addClass('brkts-popup-sc')
 		:addClass(match.opponentMode == 'uniform' and 'brkts-popup-sc-uniform-match' or 'brkts-popup-sc-team-match')
-		:node(StarcraftMatchSummary.Header({match = match}))
+		:node(StarcraftMatchSummary.Header({match = match, config = props.config}))
 		:node(StarcraftMatchSummary.Body({match = match}))
 		:node(StarcraftMatchSummary.Footer({match = match, showHeadToHead = match.headToHead}))
 end
 
 StarcraftMatchSummary.propTypes.Header = {
 	match = StarcraftMatchGroupUtil.types.Match,
+	config = 'table?'
 }
 
 function StarcraftMatchSummary.Header(props)
 	DisplayUtil.assertPropTypes(props, StarcraftMatchSummary.propTypes.Header)
 	local match = props.match
 
+	local showScore = (props.config or {}).showScore
+
 	local renderOpponent = function(opponentIx)
 		local opponent = match.opponents[opponentIx]
-		return StarcraftOpponentDisplay.BlockOpponent({
-			flip = opponentIx == 1,
+		local flip = opponentIx == 1
```

Comment:
probably better to do `side=left/right` if there is a left/right context to work with (like you suggested)

Reply:
block opponent still needs the flip
and since flip is boolean i use it to determine side in line 94
also the flip is useful in line 111 to inverse the order of the table